mlvtools/export_pipeline.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+import argparse
+import glob
+import logging
+from collections import namedtuple
+from os import chmod
+from os.path import abspath, join, exists
+from os.path import realpath, dirname
+from typing import List
+
+from jinja2 import FileSystemLoader, Environment
+
+from mlvtools.cmd import CommandHelper, ArgumentBuilder
+from mlvtools.exception import MlVToolException
+from mlvtools.helper import get_git_top_dir
+from mlvtools.mlv_dvc.dvc_parser import get_dvc_dependencies
+
+ARG_IDENTIFIER = '-'
+
+logging.getLogger().setLevel(logging.INFO)
+CURRENT_DIR = realpath(dirname(__file__))
+
+PIPELINE_EXPORT_TEMPLATE_NAME = 'pipeline-export.pl'
+ConfigurableCmds = namedtuple('ConfigurableCmds', ('cmds', 'variables'))
+
+
+def write_template(output_path, template_name: str, **kwargs):
+    """
+        Write pipeline export Jinja template. The generated file is executable.
+    """
+    loader = FileSystemLoader(searchpath=join(CURRENT_DIR, '..', 'template'))
+    jinja_env = Environment(loader=loader)
+    content = jinja_env.get_template(template_name) \
+        .render(**kwargs)
+    with open(output_path, 'w') as fd:
+        fd.write(content)
```

Comment:
I would have done the rendering in the context manager. If you can't open the file the template is still rendered.

Reply:
Done